src/Cpp/1-getting-started/2-HelloFroge/Main.cpp

```diff
@@ -11,27 +11,27 @@
 using Microsoft::WRL::ComPtr;
 
 // DirectX includes
+#include <DirectXColors.h>
+#include <DirectXMath.h>
 #include <d3d11.h>
 #include <d3dcompiler.h>
-#include <DirectXMath.h>
-#include <DirectXColors.h>
-#include <wincodec.h>
 #include <fstream>
+#include <wincodec.h>
 
+#include <iostream>
 #include <string>
 #include <string_view>
-#include <iostream>
-#include <vector>
 #include <unordered_map>
+#include <vector>
 
 #pragma comment(lib, "d3d11.lib")
 #pragma comment(lib, "dxgi.lib")
 #pragma comment(lib, "d3dcompiler.lib")
 #pragma comment(lib, "winmm.lib")
 #pragma comment(lib, "dxguid.lib")
 
-template<UINT TDebugNameLength>
-inline void SetDebugName(_In_ ID3D11DeviceChild* deviceResource, _In_z_ const char(&debugName)[TDebugNameLength])
+template <UINT TDebugNameLength>
```

Comment:
Is that a general thing space after `template`? 
I kinda like it without space, similar to c# ... `Foobar<T>`

Reply:
I think I prefer it with the space in the declaration. The usage is still `Foobar<T>`. But sure, I'll remove the space.